client/settings/express-checkout/style.scss

```diff
@@ -86,24 +112,50 @@
 				margin-bottom: 4px;
 			}
 
+			&__label-mobile {
+				display: none;
+				@include breakpoint( '<660px' ) {
+					display: block;
+					align-self: flex-end;
+				}
+			}
+
+			&__label-desktop {
+				display: none;
+				@include breakpoint( '>660px' ) {
```

Comment:
Initial look for this sounds like a hack as it has `display: none` here, and then have two `div`s, one for `__label-mobile` and another for `__label-desktop`. But I understand that with the current difference between mobile and desktop this is sometimes needed. 

Ideally, we have a way to just use CSS to overcome this, but otherwise, this is acceptable.